client/src/components/commentSider/help/Help.tsx

```diff
@@ -9,10 +9,14 @@ interface HelpCommentProps {
 export function HelpComment({ children }: HelpCommentProps) {
   return (
     <div className="help-card">
-      <div>
-        <Icon.Info size={"1rem"} />
-      </div>
-      {children}
+      <span>
+        <span>
+          <b>
+            {'Note: '}
```

Comment:
Should we replace `Note: ` with perhaps a prop called `title`? In the "Help" filter, it seems unnecessary to have `Note:`.
![image](https://user-images.githubusercontent.com/41026856/226531789-b42d24f8-f245-4db1-9fcf-a411620b1a0a.png)

Reply:
Removed `Note:`

Reply:
Will work on making the help widget look better in another PR.